README.md

```diff
@@ -238,73 +223,82 @@ Generic name of the application (e.g. `Text Editor`), used in the [`GenericName`
 Type: `String`
 Default: `package.description`
 
-Short description of the application, used in the [`Description` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description).
-
-#### options.productDescription
-Type: `String`
-Default: `package.productDescription || package.description`
-
-Long description of the application, used in the [`Description` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description).
+Short description of the application, used in the [`Comment` field of the `desktop` specification](http://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html).
 
-#### options.version
+#### options.branch
 Type: `String`
-Default: `package.version`
+Default: `master`
 
-Version number of the package, used in the [`Version` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version).
+Release branch of the flatpak, used in the [`branch` field of a flatpak-builder manifest](http://flatpak.org/flatpak/flatpak-docs.html#flatpak-builder).
 
-#### options.revision
+#### options.base
 Type: `String`
-Default: `package.revision`
+Default: `io.atom.electron.BaseApp`
 
-Revision number of the package, used in the [`Version` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Version).
+Base app to use when building the flatpak, used in the [`base` field of a flatpak-builder manifest](http://flatpak.org/flatpak/flatpak-docs.html#flatpak-builder).
 
-#### options.section
+#### options.baseVersion
 Type: `String`
-Default: `"utils"`
+Default: `master`
 
-Application area into which the package has been classified, used in the [`Section` field of the `control` specification](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Section).
+Base app version, used in the [`base-version` field of a flatpak-builder manifest](http://flatpak.org/flatpak/flatpak-docs.html#flatpak-builder).
 
-You can read more about [sections](https://www.debian.org/doc/debian-policy/ch-archive.html#s-subsections), and also check out the [list of existing sections in Debian unstable](https://packages.debian.org/unstable/).
 
-#### options.priority
+#### options.baseFlatpakref
 Type: `String`
-Default: `"optional"`
+Default: `FIXME`
```

Comment:
Same as baseFlatpakref in `flatpak-bundler`, a url or local path to a flatpakref file. So we can autoinstall the electron base app.